build/scripts/build.ps1

```diff
@@ -29,6 +29,7 @@ param (
     [switch]$packAll = $false,
     [switch]$binaryLog = $false,
     [switch]$deployExtensions = $false,
+    [switch]$launch = $false,
```

Comment:
Did you intend for `$launch` to be valid without `$deployExtensions`?

Reply:
💭 It seemed reasonable but could go either way.